curve_fitting.ipynb

```diff
@@ -64,9 +64,12 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.        xfit = np.linspace(data.min(x),data.max(x),100)
</pre><p><code>data</code> is not defined. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UMD-ENMA165/4-21-22-Class/pull/34/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>